onnxmltools/convert/common/_apply_operation.py

```diff
@@ -377,6 +377,49 @@ def apply_softmax(scope, input_name, output_name, container, operator_name=None,
     name = _create_name_or_use_existing_one(scope, 'Softmax', operator_name)
     container.add_node('Softmax', input_name, output_name, name=name, axis=axis)
 
+def apply_softplus(scope, input_name, output_name, container, operator_name=None, alpha=None, beta=None):
+    if alpha == None:
```

Comment:
This function should be called "apply_parametric_softplus" as softplus is a different operation

Reply:
Why not. :)